lib/util/git.js

```diff
@@ -172,15 +184,47 @@ module.exports._exec = execGit
 function execGit (gitArgs, gitOpts, opts) {
   opts = optCheck(opts)
   return checkGit().then(gitPath => {
-    return execFileAsync(gitPath, gitArgs, mkOpts(gitOpts, opts))
+    return promiseRetry((retry, number) => {
+      if (number != 1) {
+        opts.log.info('pacote', 'Retrying git command: ' + gitArgs.join(' ') + ' attempt # ' + number);
```

Comment:
if you're gonna log this, I'd rather this be logged in `verbose` instead.

Reply:
if a retry is going on it is very important to know to analyze the problem.
Turning on verbose on the build server is overkill and increase the build time ...

It doesn't log on the first try so for normal user nothing added.